packages/react/src/components/ListBox/test-helpers.js

```diff
@@ -105,3 +106,5 @@ export const generateItems = (amount, generator) =>
     .map((_, i) => generator(i));
 
 export const customItemToString = ({ field }) => field;
+
+export const waitForPosition = () => act(async () => {}); // Flush microtasks. Position state is ready by this line.
```

Comment:
Just want to make a note that this comes from https://floating-ui.com/docs/react#testing and more context is here: https://github.com/floating-ui/react-popper/issues/368#issuecomment-1340413010